A/accerionsensorapi/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "accerionsensorapi"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://gitlab.com/accerion/accerionsensorapi.git", "2822d9f25341fc6ea541f947452f44bbd74b314c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd accerionsensorapi/
+mkdir build
+cd build/
+cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DACCERION_API_BUILD_DYNAMIC="ON"
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf)
+]
+
```

Comment:
Quoting from the log:
```
┌ Warning: /agent/_work/1/s/A/accerionsensorapi/build/i686-linux-gnu/28jnAgl1/destdir/lib/AccerionSensorAPI/libAccerionSensorAPI.so contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder /depot/packages/BinaryBuilder/JlR0m/src/auditor/compiler_abi.jl:222
```

```suggestion
platforms = expand_cxxstring_abis(platforms)
```